src/Resources/views/Admin/Route/Grid/Field/content.html.twig

```diff
@@ -1 +1,3 @@
-{{ data.title }}
+{% if data is not same as(null) %}
```

Comment:
Hm, might be. This is what I'm also using :) But it seems to work: https://twig.sensiolabs.org/doc/2.x/tests/null.html :)